docs/BR.md

```diff
@@ -391,6 +396,8 @@ The Definitions found in the CA/Browser Forum's Network and Certificate System S
 
 **P-Label**: A XN-Label that contains valid output of the Punycode algorithm (as defined in RFC 3492, Section 6.3) from the fifth and subsequent positions.
 
+**Primary Network Perspective**: The Network Perspective used by the CA to determine 1) its authority to issue a Certificate for the requested domain(s) or IP address(es) and 2) the Applicant's authority and/or domain authorization or control of the requested domain(s) or IP address(es). 
```

Comment:
So, I'm not sure that this does a great job of capturing what a Primary Network Perspective is, particularly when compared to other, non-primary Network Perspectives.  My understanding is that all Network Perspectives are used by CAs to determine if they should issue for domain(s) and/or IP address(es).  My understanding is that the Primary network perspective is distinct by the fact that it must pass and it is the Chief perspective that ultimately looks at its result and the results from the other Network Perspectives and then makes the decision to yay or nay issuance.

Here's my attempt at some alternative text, I don't think it's perfect because ultimately the PNP might not be the decision maker, it could be a bit of code running alongside the PNP authority checking code.  But I thought I could get the conversation moving towards some better language that perhaps makes it clearer as to what a PNP is.
```suggestion
**Primary Network Perspective**: The Network Perspective used by the CA to make the determination of 1) the CA's authority to issue a Certificate for the requested domain(s) or IP address(es) and 2) the Applicant's authority and/or domain authorization or control of the requested domain(s) or IP address(es). 
```